src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import PlainButton from '../../../../components/plain-button';
+import ArrowArcRight from '../../../../icon/fill/ArrowArcRight';
+
+import type { GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+
+import CommentForm from '../comment-form';
+
+import messages from './messages';
+import './ActivityThreadReplyForm.scss';
+
+type Props = {
+    currentUser?: User,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    onReplyCreate: Function,
+};
+
+export default function ActivityThreadReplyForm({
+    currentUser,
+    mentionSelectorContacts,
+    getMentionWithQuery,
+    onReplyCreate,
+}: Props) {
+    const [showReplyForm, setShowReplyForm] = React.useState(false);
+    const placeholder = useIntl().formatMessage(messages.replyInThread);
+
+    return showReplyForm ? (
+        <CommentForm
+            className=""
+            isOpen
+            isEditing
+            getAvatarUrl={() => Promise.resolve()}
+            // $FlowFixMe
```

Comment:
Why is this needed? Can we add a brief comment explaining the need for disabling the type check here?

Reply:
This is due to the mismatch between types
CommentForm which require `user` and the `ActivityThreadReplyForm` where it's optional.
I saw it also in `ActivityFeed`
https://github.com/box/box-ui-elements/blob/6b59624dcd33f95eb24193fce1683948f8fcf6c1/src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js#L368
and `Comment`
https://github.com/box/box-ui-elements/blob/6b59624dcd33f95eb24193fce1683948f8fcf6c1/src/elements/content-sidebar/activity-feed/comment/Comment.js#L239

As far as I can tell `user` is used only to show avatar, and I don't need it here (will add screenshot of designs).